services/provider/server/server.go

```diff
@@ -543,6 +543,10 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 		}
 	}
 	var extR []*pb.ExternalResource
+
+	var storageClassName string
+	storageClassData := map[string]string{}
```

Comment:
existing code is inlining the entries, maybe it doesn't hurt to define this map w/ length of 10?

Reply:
Doing that for a `map[string]string{}` kind of defeats the purpose of the map, that just sounds like overhead for adding any new potential fields if the CSI driver ever develops any.